faq/how-do-i-change-the-colors.md

```diff
@@ -0,0 +1,29 @@
+# How do I change the colors?
+
+This article is a stub.
+
+## Update the CSS with the new colors
+
+### Relevant files
+
+- ``frontend/style/settings/_variables.scss``
+
+
+For this step, you'll need a program that can compile SCSS files into CSS. For
+example, on Ubuntu you can use ``pysassc``:
+
+```
+sudo apt install pysassc
```

Comment:
As we are already using docker, use a Docker image instead of installing sassc locally.

For example use https://hub.docker.com/r/codycraven/sassc/
```
docker run --rm -v $(pwd):/data codycraven/sassc --style=compressed --sourcemap style/rsd.scss > static/style/rsd.scss.css
```